sonar-php-plugin/src/test/java/org/sonar/plugins/php/PhpExclusionsFileFilterTest.java

```diff
@@ -83,8 +88,36 @@ public void should_ignore_empty_path_regex() {
     assertThat(filter.accept(inputFile("vendor/some_lib.php"))).isFalse();
   }
 
+  @Test
+  public void should_ignore_php_files_with_average_line_length_over_threshold() throws IOException {
+    PhpExclusionsFileFilter filter = new PhpExclusionsFileFilter(new MapSettings().asConfig());
+    File baseDir = new File("src/test/resources/exclusions");
+    SensorContextTester context = SensorContextTester.create(baseDir);
+    DefaultInputFile testFile = setupSingleFile(baseDir, context, "excluded.php");
+    assertThat(filter.accept(testFile)).isFalse();
+
+    // ignore header comment
```

Comment:
in fact we ignore header comment not only for including good files but also excluding bad files. It's possible to have narrow header comment and then long lines, i think we should test that as well (you can edit "excluded.php" file to have many header comments and not so many code lines)